### DISCUSSION TITLE
How to handle multi-arch Dockerfile digest pinning

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#8143] [Request Help] [answered]
Hi,

what's the right way to handle docker digest pinning for multiple architectures? If I add multiple Dockerfiles e.g. `Dockerfile.amd64, Dockerfile.arm` will renovate be able to update the digest for the correct architecture?

**Comment by rarkins** [comment, marked as answer]
I'm not sure exactly how that works. However I know for library images you can reference like "amd64/node" instead of just "node".

**Reply by xoxys** [reply]
Thanks, works for me.
